docs/build-the-builder.md

```diff
@@ -0,0 +1,186 @@
+
+This tags your image with namespace you set up in your environment variables.
+
+Now you can
+
+```
+$ make builder-publish
+```
+
+which, among many other things, pushes the image up to the container registry of choice. Note that you will be pushing these changes to your test repository, in order to use this builder image to build kubevirt test operators in further steps. 
+
+### Update the rpms
+
+edit your local copy of this section of the file `hack/rpm-deps.sh`:
+
+```
+LIBVIRT_VERSION=${LIBVIRT_VERSION:-0:9.5.0-5.el9}
+QEMU_VERSION=${QEMU_VERSION:-17:8.0.0-13.el9}
+SEABIOS_VERSION=${SEABIOS_VERSION:-0:1.16.1-1.el9}
+EDK2_VERSION=${EDK2_VERSION:-0:20221207gitfff6d81270b5-9}
+LIBGUESTFS_VERSION=${LIBGUESTFS_VERSION:-1:1.48.4-4.el9}
+GUESTFSTOOLS_VERSION=${GUESTFSTOOLS_VERSION:-0:1.48.2-8.el9}
+PASST_VERSION=${PASST_VERSION:-0:0^20230818.g0af928e-4.el9}
+VIRTIOFSD_VERSION=${VIRTIOFSD_VERSION:-0:1.5.0-1.el9}
+SWTPM_VERSION=${SWTPM_VERSION:-0:0.8.0-1.el9}
+SINGLE_ARCH=${SINGLE_ARCH:-""}
+BASESYSTEM=${BASESYSTEM:-"centos-stream-release"}
+```
+
+Update the version numbers of the rpms in accordance with the latest versions observable in https://mirror.stream.centos.org/9-stream/AppStream/$ARCH/os/Packages/
+
+If you do not have a version that can be looked up in the CentOS rpm collection, the build of the bazel helper container for KubeVirt will fail. 
+
+Once you get it working, you can consider incorporating your local versions of the RPM dependencies in a PR, but careful to check for CVEs and other issues in the dependencies you specified. 
+
+### Obtain the `bazeldnf` utility
+
+If you are building the builder on Power, x86 or arm architecture you can get a pre-built binary of the `bazeldnf` utility from https://github.com/rmohr/bazeldnf/releases
+
+If you are building the builder container natively on s390x, you will need to compile it from source, which is also provided at https://github.com/rmohr/bazeldnf/releases
+
+
+Now run a script which you can name `./fix-rpm-deps.sh` to update these versions throughout the installation with bazeldnf:
+
+```
+#!/bin/bash
+
+set -ex
+
+BASESYSTEM=${BASESYSTEM:-"centos-stream-release"}
+bazeldnf_repos="--repofile rpm/repo.yaml"
+
+centos_main="
+  acl
+  curl-minimal
+  vim-minimal
+"
+centos_extra="
+  coreutils-single
+  glibc-minimal-langpack
+  libcurl-minimal
+"
+
+sandboxroot_main="
+  findutils
+  gcc
+  glibc-static
+  python3
+  sssd-client
+"
+bazeldnf fetch \
+    ${bazeldnf_repos}
+
+bazeldnf rpmtree \
+        --public --nobest \
+        --name sandboxroot_s390x --arch s390x \
+        --basesystem ${BASESYSTEM} \
+        ${bazeldnf_repos} \
+        $centos_main \
+        $centos_extra \
+        $sandboxroot_main
+
+SINGLE_ARCH=<your-system-architecture> make rpm-deps
+```
+
+where <your-system-architecture> is `amd64`, `arm64`, `ppc64le` or `s390x`
+
+## Build & push Kubevirt images to regisry and make manifests
```

Comment:
Everything in this doc seems to be specific to building and publishing the builder image up until this section - I am not sure if this section fits in with the `Build the Builder Container` title?

Reply:
the  environment variables for the successful build of KubeVirt for architectures other than x86 and arm are not in 'getting-started.md' where these make targets are otherwise documented -- should I add them there? Thanks for your advice.